src/message/messageActionSheet.js

```diff
@@ -89,6 +90,10 @@ const editMessage = async ({ message, actions }: MessageAuthAndActions) => {
   actions.startEditMessage(message.id);
 };
 
+const deleteMessage = async ({ auth, message }: AuthAndMessageType) => {
+  patchMessage(auth, '(deleted)', message.id);
```

Comment:
Why not? That is exactly the spirit of deleting a message.
I still wonder if @timabbot will suggest another way to delete a message. Maybe a deleteMessage() with DELETE HTTP method.